src/Ilios/CoreBundle/Tests/Controller/AamcMethodControllerTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+namespace Ilios\CoreBundle\Tests\Controller;
+
+use FOS\RestBundle\Util\Codes;
+
+/**
+ * AamcMethod controller Test.
+ * @package Ilios\CoreBundle\Test\Controller;
+ */
+class AamcMethodControllerTest extends AbstractControllerTest
+{
+    /**
+     * @return array
+     */
+    protected function getFixtures()
+    {
+        return [
+            'Ilios\CoreBundle\Tests\Fixture\LoadAamcMethodData',
+            'Ilios\CoreBundle\Tests\Fixture\LoadSessionTypeData'
+        ];
+    }
+
+    public function testGetAamcMethod()
+    {
+        $aamcMethod = $this->container
+            ->get('ilioscore.dataloader.aamcmethod')
+            ->getOne()['aamcMethod']
+        ;
+
+        $this->createJsonRequest(
+            'GET',
+            $this->getUrl(
+                'get_aamcmethods',
+                ['id' => $aamcMethod['id']]
+            )
+        );
+
+        $response = $this->client->getResponse();
+
+        $this->assertJsonResponse($response, Codes::HTTP_OK);
+        $this->assertEquals(
+            $aamcMethod,
+            json_decode($response->getContent(), true)['aamcMethod']
+        );
+    }
+
+    public function testGetAllAamcMethods()
+    {
+        $this->createJsonRequest('GET', $this->getUrl('cget_aamcmethods'));
+        $response = $this->client->getResponse();
+
+        $this->assertJsonResponse($response, Codes::HTTP_OK);
+        $this->assertEquals(
+            $this->container->get('ilioscore.dataloader.aamcmethod')->getAll(),
+            json_decode($response->getContent(), true)
+        );
+    }
+
+    public function testPostAamcMethod()
+    {
+        $this->createJsonRequest(
+            'POST',
+            $this->getUrl('post_aamcmethods'),
+            json_encode(
+                $this->container->get('ilioscore.dataloader.aamcmethod')
+                    ->create()['aamcMethod']
+            )
+        );
+
+        $response = $this->client->getResponse();
+        $this->assertEquals($response->getStatusCode(), Codes::HTTP_CREATED);
+        $this->assertArrayHasKey('location', $response->headers->all(), print_r($response->headers->all(), true));
+    }
+
+    public function testPostBadAamcMethod()
+    {
+        $invalidAamcMethod = array_shift(
+            $this->container->get('ilioscore.dataloader.aamcmethod')->invalid()
+        );
+
+        $this->createJsonRequest(
+            'POST',
+            $this->getUrl('post_aamcmethods'),
+            $invalidAamcMethod
+        );
+
+        $response = $this->client->getResponse();
+        $this->assertEquals($response->getStatusCode(), Codes::HTTP_BAD_REQUEST);
+    }
+
+    public function testPutAamcMethod()
```

Comment:
I thought that relationships through the owning side would be persisted by doctrine. So doesn't it depend on the data seed?